test/e2e/serial/tests/resource_hostlevel.go

```diff
@@ -113,6 +151,16 @@ var _ = Describe("[serial] numaresources host-level resources", Serial, func() {
 					corev1.ResourceMemory:           resource.MustParse("256Mi"),
 					corev1.ResourceEphemeralStorage: resource.MustParse("256Mi"),
 				},
+				corev1.PodQOSBurstable,
+			),
+			Entry("[tmscope:cnt][qos:be] with ephemeral storage",
+				intnrt.SingleNUMANode,
+				intnrt.Container,
+				// required resources for the test pod
+				corev1.ResourceList{
+					corev1.ResourceEphemeralStorage: resource.MustParse("256Mi"),
+				},
+				corev1.PodQOSBestEffort,
```

Comment:
since the tests are dealing with pod with one container, we can omit passing the tmscope. this way the test would be executed on all available configured tmscopes whenever the test is triggered.

Reply:
Ok, _really_ fixed now. Took me 3 reads to really get your point, sorry!